Questions/Q25.java

```diff
@@ -0,0 +1,37 @@
+/* Neha and Nisha are playing a game .Nisha ask Neha to enter the number N from given range of 0 to 99 .Now Nisha need to print the number till N and once it exceed the number from N then Print "Games End".If Neha enter the number less then 0 print Invalid input.
+
+Input Format
+
+Program should take the input for Number N.
+
+Constraints
+
+0>=N<100
+
+Output Format
+
```

Comment:
remove these extra lines | Follow the format of the question as in other past questions